css/index.css

```
@@ -1,3 +1,296 @@
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+.container {
+  width: 900px;
+  margin: 0 auto;
+}
+
+header {
+    display: flex;
+    flex-wrap: wrap-reverse;
+    justify-content: center;
+}
+
+header nav {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+header nav a {
+	margin: 52px 15px 0 40px;
+  color: grey;
+  font-size: 14px;
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;  
+}
+
+p {
+  line-height: 18px;
+}
+
+ul {
+  list-style-type: square;
+  list-style-position: inside;
+  line-height: 30px;
+  margin-top: 20px;
+}
+
+.logo {
+  margin: 20px 0 0 100px;
+  padding-top: 20px;
+}
+
+.hero {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+  margin: 50px 0 50px 0;
+}
+.cta {
+  text-align: center; 
+  line-height: 60px;
+}
+
+.cta h1 {
+	font-size: 72px;
+}
+
+.about-services {
+  margin: 30px 0 0 30px;
+}
+
+.about-services h1 {
+	font-size: 24px;  
+}
+
+button {
+  font-size: 16px;
+  padding: 5px 40px 5px 40px;
+  border: 1px solid black;
+}
+
+.content {
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;  
+  margin: 30px; 
+}
+
+.services-content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;  
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;  
+  margin: 30px;
+  padding-bottom: 40px;
+}
+
+.service-item {
+  margin: 40px 0 0 0;
+  padding: 20px;
+  background: lightgrey;
+  border: 1px solid black;
+  width: 400px;
+}
+
+.service-item p {
+  line-height: 18px;
+}
+.service-item button {
+  margin-top: 20px;
+  font-size: 12px;
+  padding: 7px 20px 7px 20px;
+  border: 1px solid black;  
+  border-radius: 5px;
+}
+
+.two-col {
+  display: flex;
+  justify-content: space-between;
+  margin: 40px 0 40px 0;
+}
+
+.upper-items {
+ width: 400px;
+}
+
+.three-col {
+  display: flex;
+  justify-content: space-between;  
+  margin: 40px 0 40px 0;
+}
+
+.lower-items {
+  width: 250px;
```

```suggestion
  width: 30%;
```
I am recommending the same solution for the lower items as the upper items, but with a different percentage